### DISCUSSION TITLE
How to set graph-type

### DISCUSSION THREAD

**Discussion by freedomwarrior** [akvorado/akvorado#451] [Q&A] [answered]
I noticed that in 1.6.4 was added option graph-type.
How properly to set Sankey as default options?
I try this one but it doesn't work:
```
  default-visualize-options:
    graph-type: sankey
```

**Comment by vincentbernat** [comment, marked as answer]
It should work. I have just tested it. Be sure to click on the visualize tab, not just reload the current page. Also, be sure to restart both orchestrator and console if you use docker, as the configuration to the console component is provided by the orchestrator.

**Reply by freedomwarrior** [reply]
Yep, it works.
I forgot to restart both services.
